python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1003,6 +1014,7 @@ def _impl(inputs, attr, params):
     'Sigmoid'                           : AttrCvt('sigmoid'),
     'Fill'                              : _fill(),
     'GatherV2'                          : _gather_v2(),
+    'Gather'                            : _gather_v1(),
```

Comment:
how about mapping to `_gather_v2`?

Reply:
it doesn't work, gather and gatherv2 def not the same, for example axis is not is gather

Reply:
got it. thank you!